packages/dapp/public/index.html

```diff
@@ -12,6 +12,11 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>ORGiD profile manager</title>
+    <style>
```

Comment:
It is not acceptable to add stylings into an index.html directly.
The project must have a common `reset.css` with base stylings. Such a file can be created and integrated manually or using UI framework.

Reply:
```
body {
        margin:0;
      }
```
still in the index.html but must be removed. Please move global stylings to the separate file